### ISSUE TITLE
[24H2] The "disable rounded corner" option does not work

### ISSUE THREAD

**Issue by Michellehdr** [valinet/ExplorerPatcher#3555]
ExplorerPatcher version: 22621.3880.66.4
Windows Version: Windows insider build

Repro Steps:
1. Deploy latest OS build.
2. Perform Windows Updates.
3. Install the application from vendor site.
4. Launch the application.
5. Click on Other option.
6. Check/uncheck the Disable Rounded Corner and Disable quadrants when snapping.
7. Observe that automatically unchecking the disable rounded corner option checking the Disable quadrants when snapping.

Actual result:
The "disable rounded corner" option will be automatically unchecked after checking the "Disable quadrants when snapping" option.

**Comment by Michellehdr** [comment]
@Amrsatrio @pyrates999, could anyone please take a look at this bug?

**Comment by pyrates999** [comment]
As I'm not a developer, I'll let one of the developers here respond.

But I think we need the insider build available to test from in order to look into why this happened.

**Comment by Michellehdr** [comment]
@pyrates999 do you know who is the developer? Can we tag him here?

**Comment by Michellehdr** [comment]
@valinet could you please take a look at this bug? Thanks!

**Comment by wikiepeidia** [comment]
it also happened to LTSC 2024 26100.1301 on explorer patcher v22621.3880.66.6, more likely due to incompabilities

**Comment by EntropyGenerator** [comment]
Same behavior at LTSC 2024 26100.1742, ep version 22621.3880.66.6

**Comment by mikrrice** [comment]
Same issue. Manually starting the ep_dwm service or exe both stop instantly. LTSC 2024 26100.2033, EP ver. 22621.3880.66.7

**Comment by kainblock** [comment]
It works on me. Did they fix it ? 
![image](https://github.com/user-attachments/assets/135e220b-c84e-44af-8d2d-6919d4a0f369)

![image](https://github.com/user-attachments/assets/3aeea2af-0d04-436a-a4f3-4bce8def66b5)

**Comment by Amrsatrio** [comment]
MS did. It's a miracle then.

**Comment by HowXu** [comment]
I'm sorry to bother you guys. I would like to know if there is any way to make Disable round corners work at present. The windows seems too ugly

**Comment by pyrates999** [comment]
I'm going to post this here as I remember it being posted before: https://github.com/valinet/ep_dwm

Referenced here: https://github.com/valinet/ExplorerPatcher/discussions/291#discussioncomment-2062300

**Comment by Ev3Friv** [comment]
> I'm sorry to bother you guys. I would like to know if there is any way to make Disable round corners work at present. The windows seems too ugly

I used this: https://github.com/oberrich/win11-toggle-rounded-corners

I'm running windows 11 pro 24h2
Version	10.0.26100 Build 26100

I was sceptical at first so tried it on a vm and it worked fine, I eventually tried it on my host machine and it works perfectly.
Hope this helps you.

**Comment by JPHFanedits** [comment]
just updated to Windows 11 Pro 24H2 26100.2605 and rounded corners option automatically resets when file explorer is reset and it never sharpens the corners

**Comment by HaloRage** [comment]
@valinet please check this this makes my os look like windows 11 and 10 mixed :(

**Comment by tuaris** [comment]
Yeah, this isn't working at all for me on 10.0.26100.1742 with [22621.4317.67.1](https://github.com/valinet/ExplorerPatcher/releases/tag/22621.4317.67.1_b93337a)

Damn Microsoft always wanting to cut corners.

**Comment by jay-ramani** [comment]
> I reinstalled Windows just to downgrade to 23H2 so that I could disable rounded corners again. At least it is supported until November 2025.
> 
> Ironically most (modern) Linux desktop environments have rounded corner themes as well!

Use XFCE

**Comment by HowXu** [comment]
> > > I'm sorry to bother you guys. I would like to know if there is any way to make Disable round corners work at present. The windows seems too ugly
> > 
> > 
> > I used this: https://github.com/oberrich/win11-toggle-rounded-corners
> > I'm running windows 11 pro 24h2 Version 10.0.26100 Build 26100
> > I was sceptical at first so tried it on a vm and it worked fine, I eventually tried it on my host machine and it works perfectly. Hope this helps you.
> 
> Worked like a charm. Thank you. Although, would be good to see this fixed with Explorer Patcher. Poor guys, they have to keep fixing Microsoft's so called "features"!

Thanks very much. It work well . Thank you again !!!!!!!!!!

**Comment by michalschovanek** [comment]
> > I'm sorry to bother you guys. I would like to know if there is any way to make Disable round corners work at present. The windows seems too ugly
> 
> I used this: https://github.com/oberrich/win11-toggle-rounded-corners
> 
> I'm running windows 11 pro 24h2 Version 10.0.26100 Build 26100
> 
> I was sceptical at first so tried it on a vm and it worked fine, I eventually tried it on my host machine and it works perfectly. Hope this helps you.

This works perfectly in Windows 11 24H2. Thank you very much! I hope they can implement something similar in Explorer Patcher.

**Comment by Amrsatrio** [comment]
Fixed in 68.2.
